crypto/digestible/derive/test/tests/behavior.rs

```diff
@@ -67,38 +42,81 @@ struct GenericFoo<X: Digestible> {
 #[test]
 fn foo1() {
```

Comment:
It would be nice to have a one-line comment about what property this test should check. It seems to be something along the lines of, "A struct that only contains primitive fields should yield an AST containing an aggregate node with one primitive node for each field; and the instance should digest to some known bytes."

Writing that out, it feels like this is testing several different-but-related pieces of functionality, so maybe tests like this should be separated into testing the AST, and testing test vectors of digests. Just food for thought.